homeassistant/helpers/entity.py

```diff
@@ -1289,7 +1289,12 @@ def _async_subscribe_device_updates(self) -> None:
             self.async_on_remove(self._async_unsubscribe_device_updates)
 
     def __repr__(self) -> str:
-        """Return the representation."""
+        """Return the representation.
+
+        If the entity is not added to a platform it's not safe to call _stringify_state.
+        """
+        if self._platform_state != EntityPlatformState.ADDED:
+            return f"<entity {self.entity_id}={STATE_UNKNOWN}>"
```

Comment:
The entity_id is not valid if the entity is not added. We should maybe rewrite it?

Reply:
entity unknown.unknown=unknown

😆

I guess it's better than nothing.